lib/decorators/BaseDecorator.cpp

```diff
@@ -47,7 +47,7 @@ namespace MAT_NS_BEGIN {
 #else
         record.extSdk[0].libVer = PAL::getSdkVersion();
 #endif
-        auto sessionData = m_owner.GetLogSessionData();
+        const auto* sessionData = m_owner.GetLogSessionData();
```

Comment:
Does this changing the type of `sesstionData` (removing pointer), and is this expected?

Reply:
Ok, seems the variable type is not change, it is just required to be a pointer.

Reply:
Correct, this has no change to the type but it's cv qualifiers.